csv2docx/cli.py

```diff
@@ -1,7 +1,22 @@
 from mailmerge import MailMerge
 import click
 import csv
-import sys
+import pathlib
+
+
+def create_path_if_not_exists(path: str) -> pathlib.Path:
```

Comment:
Not a fan of the name of this method ... open to suggestions

Reply:
Also, the problem is that the parameter `path` can be a directory `output`, an absolute path `/user/bin/output` or relative path `../../output` and we should except all those conditions.

Rather than deal with edge cases, I used pathlib as it converts the given path to a `concrete path` that is user platform specific 👍

Reply:
Perhaps go with 
`def concrete_path(path: str) -> pathlib.Path:`
with a note that this method will create the directory if non-existent. 

Ps. does it handle a nested non-existent directory?

Reply:
Great suggestion re:name 👍 

Yes -- it handles nested non-existent directories thanks to the `parents` option.

Reply:
How about `absolute_path` as it's more common terminology?